docs/topics/logging.rst

```diff
@@ -238,9 +238,10 @@ scrapy.utils.log module
 
 .. autofunction:: configure_logging
 
-    ``configure_logging`` is automatically called when using Scrapy commands,
-    but needs to be called explicitly when running custom scripts. In that
-    case, its usage is not required but it's recommended.
+    ``configure_logging`` is automatically called when using Scrapy commands
+    or :class:`scrapy.crawler.CrawlerProcess`, but needs to be called explicitly
```

Comment:
:class:`scrapy.crawler.CrawlerProcess`

renders as `scrapy.crawler.CrawlerProcess`. Could you use [`~`](http://www.sphinx-doc.org/es/stable/markup/inline.html#cross-referencing-syntax) to make it just `CrawlerProcess`?

    :class:`~scrapy.crawler.CrawlerProcess`

It’s a shorthand for:

    :class:`CrawlerProcess <scrapy.crawler.CrawlerProcess>`

This applies to the `CrawlerRunner` reference as well.

Reply:
Done. Kindly check the latest force push to see the changes. Thanks!